packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.types.ts

```diff
@@ -389,6 +393,11 @@ export interface IContextualMenuItem {
     primaryDisabled?: boolean
   ) => IMenuItemClassNames;
 
+  /**
+   * Call to provide customized styling that will layer on top of the variant rules
+   */
+  styles?: IStyleFunctionOrObject<IContextualMenuItemStyleProps, IContextualMenuItemStyles>;
```

Comment:
We spoke in-person -- I'm going to replace this `styles?` with the props bag `itemProps?: IContextualMenuItemProps` so that we don't have to change API signature in the future 👍. Posting iteration shortly.

Reply:
Done in https://github.com/OfficeDev/office-ui-fabric-react/pull/6094/commits/8f98b2f50c8a9693b95223534dc2118686069f52